### ISSUE TITLE
Cant transfer my xrp out of this broken wallet

### ISSUE THREAD

**Issue by seanobj** [sologenic/sologenic-expo-wallet#154]
This is completely disgraceful. Cant create a Solo wallet, cant even transfer my XRP back out of this trash app. Devs should be ashamed at best and held accountable at worst. Where is the support???

**Comment by joshmcl8** [comment]
I am in the same boat. Feeling pretty let down right now…

**Comment by kupiV** [comment]
I can’t send XRP out -
Error - can’t queue

**Comment by joshmcl8** [comment]
So do the devs ever reply on here? When will we know when the fault gets rectified?

**Comment by paintherapys** [comment]
I totally agree with you and have same issues. I would appreciate an answer from the devs but nothing yet after 2-3 days

**Comment by Willyco89** [comment]
It is totally decentralized so the money is lost

**Comment by joshmcl8** [comment]
Pretty sure once the technical issue is resolved you will be able to send the tokens out of the wallet. They haven’t disappeared :/

**Comment by michaelmcfann26** [comment]
Yeah having same problem haven’t been able to do anything on the solo wallet on my phone or on the desktop version this is bullshit. Can Sologenic team respond what is going on when will the issues be fixed!

**Comment by michaelmcfann26** [comment]
Hello SOLOGENIC IS ANYONE OUT THERE?

**Comment by sologenic** [comment]
We have released a fix. Please download the latest version from Apple Store or Google Play store.

**Comment by michaelmcfann26** [comment]
Download the latest Sologenic app?

**Comment by michaelmcfann26** [comment]
There is only 1 app and I already have it

**Comment by michaelmcfann26** [comment]
![Uploading CDE17D0F-4DD0-4C4C-B0CF-6D2A9A67F4FC.png…]()

**Comment by michaelmcfann26** [comment]
They fixed it!!! I can send out my Xrp!!!!!

**Comment by michaelmcfann26** [comment]
Everyone you should have an update on your solo app! Thank you Sologenic team! Much appreciated!
